File: Bindings~/java/src/main/java/ro/altom/altunitytester/Notifications/AltUnityLogNotificationResultParams.java
package ro.altom.altunitytester.Notifications;

public class AltUnityLogNotificationResultParams {
    
    public String message;
    public String stackTrace;
    public int level;

    public AltUnityLogNotificationResultParams(String message, String stackTrace, int level) {
        this.message = message;
        this.stackTrace = stackTrace;
        this.level = level;
    }
}

File: Assets/AltUnityTester/AltUnityDriver/Notifications/AltUnityLogNotificationResultParams.cs
using Altom.AltUnityDriver.Logging;

namespace Altom.AltUnityDriver.Notifications
{
    public class AltUnityLogNotificationResultParams
    {
        public string message;
        public string stackTrace;
        public AltUnityLogLevel level;

        public AltUnityLogNotificationResultParams(string message, string stackTrace, AltUnityLogLevel level)
        {
            this.message = message;
            this.stackTrace = stackTrace;
            this.level = level;
        }
    }
}